### DISCUSSION TITLE
didChageConfiguration

### DISCUSSION THREAD

**Discussion by XXiaoA** [williamboman/nvim-lsp-installer#473] [Q&A] [answered]
how to send a didChageConfiguration request? I want to configure `pylsp`, but i need to send a `didChageConfiguration`. So how can i do that?

**Comment by williamboman** [comment, marked as answer]
Hello! lspconfig sends this [request for you](https://github.com/neovim/nvim-lspconfig/blob/531751ef03d77bf094f1fc07844b6c03ec4862bb/lua/lspconfig/configs.lua#L174-L176) upon initialization. It'll use the `settings` table provided to the `setup()` call (search `:h lspconfig` for `didChangeConfiguration` for more info.

If you want to actually send this request manually for some reason, I'd suggest reading the lsp docs (`:h lsp`) for guidance!

**Comment by XXiaoA** [comment]
okay，thanks
